deploy/helm-chart/README.md

```diff
@@ -132,25 +132,28 @@ kubectl delete -f https://github.com/kubernetes-sigs/gateway-api/releases/downlo
-| `service.type`                       | The type of service to create for the NGINX Kubernetes Gateway.                                                                                                                                              | Loadbalancer                                                                                                    |
-| `service.externalTrafficPolicy`      | The `externalTrafficPolicy` of the service. The value `Local` preserves the client source IP.                                                                                                                | Local                                                                                                           |
-| `service.annotations`                | The `annotations` of the NGINX Kubernetes Gateway service.                                                                                                                                                   | {}                                                                                                              |
-| `service.ports`                      | A list of ports to expose through the NGINX Kubernetes Gateway service. Update it to match the listener ports from your Gateway resource. Follows the conventional Kubernetes yaml syntax for service ports. | [ port: 80, targetPort: 80, protocol: TCP, name: http; port: 443, targetPort: 443, protocol: TCP, name: https ] |
-| `metrics.disable`                    | Disable exposing metrics in the Prometheus format.                                                                                                                                                   |false                                                                                                              |
-| `metrics.port`                       | Set the port where the Prometheus metrics are exposed. Format: [1024 - 65535]                                                                                                                                  |9113                                                                                                              |
-| `metrics.secure`                     | Enable serving metrics via https. By default metrics are served via http. Please note that this endpoint will be secured with a self-signed certificate.                                                                                                              |false                                                                                                              |
+| Parameter | Description | Default Value |
```

Comment:
I think our IDEs format the tables differently. My [PR](#1048) will reformat the table to look like the previous version

Reply:
Oh that's a bummer. I don't have any settings in my IDE for this, so adding new fields is a pain because of how large the spacing is. I manually made these changes.